VARIABLES.md

```diff
@@ -60,3 +60,9 @@
 | s3_lifecycle_policy_transition_period | Number of days for transition to a different storage class using lifecycle policy. | string  | `30` | no |
 | s3_storage_class | Destination S3 storage class for transition in the lifecycle policy. | string  | `INTELLIGENT_TIERING` | no |
 | vpc_id | VPC ID. | string | - | yes |
+| ami_id | Amazon Linux AMI. | string | - | no |
```

Comment:
maybe we get latest Amazon AMI using data source otherwise users can choose centos, debian, redhat and packages maynot work as expected?

Reply:
we are already using datasource to get latest AMI, this is to override default image.